packages/features/tasker/tasks/sendEmail.ts

```diff
@@ -8,9 +8,11 @@ const sendEmailPayloadSchema = z.object({
   payload: z.string(),
 });
 
+type SendEmailPayload = z.infer<typeof sendEmailPayloadSchema>;
+
 export async function sendEmail(payload: string): Promise<void> {
   try {
-    const parsedPayload = sendEmailPayloadSchema.parse(JSON.parse(payload));
+    const parsedPayload: SendEmailPayload = sendEmailPayloadSchema.parse(JSON.parse(payload));
```

Comment:
Hey @anikdhabal as far as I know, parsedPayload already has the correct type inference. These typing should be added to the packages/features/tasker/tasker.ts typings in order to keep the task creation type safe. Not in here. Will close the PR for now. 🙏